xwiki-platform-core/xwiki-platform-mentions/xwiki-platform-mentions-test/xwiki-platform-mentions-test-pageobjects/src/main/java/org/xwiki/mentions/test/po/MentionNotificationPage.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package org.xwiki.mentions.test.po;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.xwiki.test.ui.po.BaseElement;
+
+import static org.openqa.selenium.By.cssSelector;
+import static org.openqa.selenium.By.xpath;
+
+/**
+ * Mentions notifications page object.
+ *
+ * @version $Id$
+ * @since 12.7RC1
+ */
+public class MentionNotificationPage extends BaseElement
+{
+    private static final String NOTIFICATION_EVENT_SELECTOR = "div.notification-event";
+
+    private static final String GROUP_SELECTOR =
+        "button.toggle-notification-event-details";
+
+    private static final String TEXT_SELECTOR = "tr td.description";
+
+    private static final String EMITTER_SELECTOR =
+        "tr td span.notification-event-user";
+
+    private static final String SUMMARY_SELECTOR = "../following-sibling::tr/td/blockquote";
+
+    /**
+     * Opens a notification group.
+     * @param notificationNumber The notification index.
+     */
+    public void openGroup(int notificationNumber)
```

Comment:
It doesn't look related to Mention to me, sounds like a general need for Notifications test instead.

Reply:
It's same for the other methods: they are all general to notifications apparently. Maybe you could introduce a GroupedNotificationElement PO for all of them, but shouldn't be tied to mention.